contracts/test/EIP1820.js

```diff
@@ -0,0 +1,40 @@
+
+describe('EIP 1820', () => {
+  if (isGwMainnetV1()) {
+    return;
+  }
+
+  let provider = waffle.provider;
+
+  describe('Deploy', () => {
+    it('transferred value and fee', async () => {
+      let code = await provider.getCode(contract);
+      expect(code).to.be.equal("0x", "account isn't deployed");
+      // Create a transaction object
+      let tx = {
+        to: deployer,
+        // Convert currency unit from ether to wei
+        value: ethers.utils.parseEther("0.08")
+      };
+      // Send a transaction
+      let txObj = await wallet.sendTransaction(tx);
```

Comment:
Where the `wallet` from ?

Reply:
It is an error due to copy-paste, how to call `sendTransaction` in hardhat?

Reply:
`signer.sendTransaction` or `provider.sendTransaction`
- https://github.com/nervosnetwork/godwoken-tests/blob/dabafec7db72153ca1dcd333bf3ecdc9d8eb5d76/contracts/test/IntrinsicGas.js#L85
- https://github.com/nervosnetwork/godwoken-tests/pull/172/files#diff-e30d120ce1dbf7897752b9f299aee8e8356eeb2e7a13d938d71059966d7f974dR59